common/scummsys.h

```diff
@@ -135,6 +135,7 @@
 	#include <stddef.h>
 	#include <assert.h>
 	#include <ctype.h>
+	#include <stddef.h>
```

Comment:
This is another problem with having out of tree / branched engines for too long, especially when they have commits changing common / graphics / video code :/